### PR TITLE
feature(article): Add sections field, unstitch Positron 

### PR DESCRIPTION
**Pull request by damassi**
This updates the article type with a new `sections` field and removes positron stitching (since it doesn't work with anonymous types):

```graphql
{
  article(id: "artsy-editorial-detroit-gallery-library-street-collective-supporting-artists-connecting-community") {
    internalID
    sections {
      ... on ArticleSectionText {
        type
        body
      }
      ... on ArticleSectionEmbed {
        type
        url
        height
        mobile_height
        layout
      }
      ... on ArticleSectionCallout {
        type
      }
      ... on ArticleSectionImageCollection {
        type
        layout
      }
      ... on ArticleSectionImageSet {
        type
        title
        layout
      }
      ... on ArticleSectionSocialEmbed {
        type
        url
        layout
      }
    }
  }
}
```

Screen Shot 2022-01-06 at 10 46 15 PM

cc @artsy/grow-devs

### CODE REVIEW

**Comment by dzucconi** [comment]
`EditorialObjectOrObjectOrObjectOrObjectOrObjectOrObjectOrObjectOrObject` lol

File: src/schema/v2/article/sections/ArticleSectionCallout.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    thumbnail_url: {
+      type: GraphQLString,
+    },
+    text: {
+      type: GraphQLString,
+    },
+    article: {
+      type: GraphQLString,
+    },
+    hide_image: {
```
Comment (dzucconi) [inline src/schema/v2/article/sections/ArticleSectionCallout.ts:25]: Might as well camel case these now?

Comment (damassi) [reply src/schema/v2/article/sections/ArticleSectionCallout.ts:25]: Woke up remembering this! Will do

**Review by dzucconi — APPROVED** [review]
👍 Good to merge?

**Pull request merged**